### PR TITLE
ref(integrations): Cleaning up the Notification Settings

### PR DESCRIPTION
**Pull request by mgaeta**
Miscellaneous code clean-ups that should make the following PRs simpler: https://github.com/getsentry/sentry/pull/24258 https://github.com/getsentry/sentry/pull/24325

### CODE REVIEW

File: src/sentry/models/organizationaccessrequest.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def send_request_email(self):
 
         msg.send_async([m.user.email for m in member_list])
 
+    #
```
Comment (ceorourke) [inline src/sentry/models/organizationaccessrequest.py:69]: what's this for?

Comment (mgaeta) [reply src/sentry/models/organizationaccessrequest.py:69]: That shouldn't be there, let me clean up some weirdness in a new commit.

File: tests/sentry/api/endpoints/test_user_notification_fine_tuning.py [hunk 2]
```diff
@@ -240,82 +170,54 @@ def test_saves_and_returns_deploy(self):
             == "0"
         )
 
-        update = {}
-        update[self.org.id] = 1
-        resp = self.client.put(url, data=update)
+        self.get_valid_response("me", "deploy", method="put", status_code=204, **{self.org.id: 2})
```
Comment (ceorourke) [inline tests/sentry/api/endpoints/test_user_notification_fine_tuning.py:173]: it looks like the org ID is 1 here, not 2

**Review by ceorourke — APPROVED** [review]
Looks good now to my eyeballs, since it's a big change might wanna get another set too 👀

File: tests/sentry/api/endpoints/test_user_notification_fine_tuning.py (hunk 2 above)
Comment (mgaeta) [reply tests/sentry/api/endpoints/test_user_notification_fine_tuning.py]: Good catch 👍🏽

**Pull request merged**